src/test/kotlin/minesweeper/domain/MineBoardTest.kt

```diff
@@ -0,0 +1,40 @@
+package minesweeper.domain
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.shouldBe
+import minesweeper.application.MineSweeperService
+import minesweeper.fixture.TestMineBoardCreateStrategy
+import minesweeper.request.MineBoardCreateRequest
+
+class MineBoardTest : FunSpec({
+    val service = MineSweeperService(mineBoardCreateStrategy = TestMineBoardCreateStrategy)
+
+    test("지뢰찾기 보드의 각 좌표 정보는 인접한 8개의 좌표의 지뢰 갯수만큼 카운팅된다.") {
+        val request = MineBoardCreateRequest(height = 3, width = 3, mineCapacity = 5)
+        val inputLines = listOf(
+            listOf(SymbolType.ZERO, SymbolType.MINE, SymbolType.ZERO),
+            listOf(SymbolType.ZERO, SymbolType.ZERO, SymbolType.ZERO),
+            listOf(SymbolType.MINE, SymbolType.ZERO, SymbolType.MINE)
+        )
```

Comment:
아래 처럼 지뢰 판을 주석의 도움을 받으면 더 가독성을 높일 수 있어요!
```suggestion
        /**
         *  0 X 0
         *  0 0 0
         *  X 0 X
         */
        val inputLines = listOf(
            listOf(SymbolType.ZERO, SymbolType.MINE, SymbolType.ZERO),
            listOf(SymbolType.ZERO, SymbolType.ZERO, SymbolType.ZERO),
            listOf(SymbolType.MINE, SymbolType.ZERO, SymbolType.MINE)
        )
```